addon/helpers/build-secondary-nav-links.js

```diff
@@ -0,0 +1,83 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+/**
+ * @module ember-osf
+ * @submodule helpers
+ */
+
+/**
+ * Returns secondary links corresponding to a given service
+ *
+ * @class buildSecondaryNavLinks
+ * @param {String} currentService name of current service (HOME, PREPRINTS, REGISTRIES, or MEETINGS)
+ * @param {Object} session Injected session
+ * @return {Array} Returns array of secondary link information
+ */
+export function buildSecondaryNavLinks(params/*, hash*/) {
+    const currentService = params[0].toUpperCase();
+    const session = params[1];
+    const osfUrl = config.OSF.url;
+    let links = Ember.Object.create({
+        HOME: [
+            {
+                name: `${session.isAuthenticated ? 'My Projects' : 'Browse'}`,
```

Comment:
`session.isAuthenticated` [appears to be](https://github.com/simplabs/ember-simple-auth/blob/master/addon/services/session.js) a computed property. Would it make more sense to fetch it as `session.get('isAuthenticated')`, to ensure that we are getting the [calculated result](http://emberjs.com/api/classes/Ember.Service.html#method_get) (rather than just the existence) of the property?